content/about-us/involved/index.en.md

```diff
@@ -8,28 +8,47 @@ title: "Get Involved"
-- Content along with recordings for our past events are openly accessible over [YouTube](https://www.youtube.com/@RLadiesGlobal) channel, reach out to speakers that you like and let them know that you appreciate their content
+These are some initial steps you can take to become part of the R-Ladies community:
 
-#### 💡Always room for more 
+- **Update Your Directory Profile:** Make sure your details are current in the [R-Ladies directory](/directory) so others can connect with you.
+- **Join Our Community Slack:** Engage in discussions, ask questions, and stay informed by joining our [community slack channel](https://rladies.org/form/community-slack).
+- **Follow Us on Social Media:** Keep up with the latest news, events, and activities by following us on [social media](https://hachyderm.io/@RLadiesGlobal).
+- **Connect with Your Local Chapter:** Join your regional [R-Ladies chapter](https://www.meetup.com/pro/rladies/) or the remote chapter to connect with like-minded individuals in your area or online.
+- **Attend Events:** Participate in online and in-person events to learn, network, and engage with the community. Event details are usually shared on our Slack channels and social media.
+- **Explore Past Event Content:** Our past event recordings and materials are openly accessible on our [YouTube](https://www.youtube.com/@RLadiesGlobal) channel. Feel free to reach out to speakers whose content you appreciate!
 
-- Propose a talk for a meetup/event/conference 
-- Become a chapter organizer by volunteering with your local chapter or starting your own (please get in touch via [chapters@rladies.org](mailto:chapters@rladies.org))
-- Contribute a blog post to the [website](https://rladies.org/), we are always here to help. 
+### ✨ Making a Difference
 
-#### 🤝 Above and beyond 
+Here are ways you can contribute your skills and time as an individual:
 
-- Join the [global team](https://rladies.org/about-us/global-team/), help us create a diverse space 
-- Create issues or PR(s) over [GitHub](https://github.com/rladies) for our [website](https://github.com/rladies/rladies.github.io) & [guide](https://github.com/rladies/rladiesguide) when you encounter sections which could be improved
-- Contribute to community discussion and share your thoughts
-- Support your local chapters with venue space and funding for food and beverages
-- Support our infrastructure and behind-the-scenes operations. We appreciate your donations via [PayPal](https://www.paypal.com/donate/?hosted_button_id=VB4EK67MJ373E)
+- **Propose a Talk:** Share your knowledge and passion by proposing a talk for a local meetup, event, or conference.
+- **Contribute to Our Blog:** Share your insights, tutorials, or experiences by writing a blog post for the [website](https://rladies.org/). We're always happy to help you develop your ideas.
+- **Engage in Community Discussions:** Share your thoughts, answer questions, and contribute to the vibrant discussions within our community.
```

Comment:
is the intent here for slack community discussions? or generally something not platform specific?

Reply:
right, I havent really thought this through properly. I was thinking engaging in general, I think. Engage with us on social media and slack etc. Need to think a bit about this...